app/views/casa_org/edit.html.erb

```diff
@@ -1,60 +1,60 @@
-<h1><%= t(".title") %></h1>
+<h1>Editing CASA Organization</h1>
 
 <div class="card card-container">
   <div class="card-body">
     <%= form_with(model: current_organization, local: true) do |form| %>
       <%= render "/shared/error_messages", resource: current_organization %>
       <div class="field form-group">
-        <%= form.label :name, t("common.name") %>
```

Comment:
I haven't removed the keys for the common translations as they're still being referenced by other pages, which are out of scope of this change